x/staking/keeper/cons_pubkey.go

```diff
@@ -90,3 +142,61 @@ func bytesSliceExists(sliceList [][]byte, targetBytes []byte) bool {
 	}
 	return false
 }
+
+// UpdateAllMaturedConsKeyRotatedKeys udpates all the matured key rotations.
+func (k Keeper) UpdateAllMaturedConsKeyRotatedKeys(ctx sdk.Context, maturedTime time.Time) error {
```

Comment:
Sounds like this should be `PurgeAllMaturedConsKeyRotatedKeys`

Reply:
We are using this method in the tests